Gemfile

```diff
@@ -39,6 +39,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails'
   gem 'factory_bot_rails'
+	gem 'database_cleaner'
```

Comment:
alignment

Reply:
use 2 spaces instead of tabs
and tell it to all team members